### ISSUE TITLE
Copy last spoken phrase or sentence

### ISSUE THREAD

**Issue by balajip881** [nvaccess/nvda#14752]
### Is your feature request related to a problem? Please describe.
In talk back, android os ,  I can copy last spoken phrase  very easily with a gesture. I need the same in nvda with a key board  shortcut.

In the interview, i need to copy chat message in chat, in zoom web app . in focus mode nvda is telling chat message but in browse mode nvda is not telling the message. 

In teams app also i faced thed similar type of issue. i need to copy  message of many lines. in browse mode i tried. but nvda is  not showing full message so failed to do so. in focus mode i pressed shift and down, but it resulted in only oneline without any new lines. that is not copying 
\n .

### Describe the solution you'd like

Need to add a key board shortcut for copying last spoken phrase or sentence.
It is already implemented in talk back latest version in android

**Comment by Adriani90** [comment]
In teams you can copy the whole Message in browse mode, but but you habe to navigate below the heading of each message. There is a button after the heading which expands a bar with two buttons. You can then expand the chat to a dedicated window after pressing "more options" button.

This issue would be solved if the speech viewer was an accessible window having kind of a history feature, let's say of the last 5 spoken strings.

**Comment by balajip881** [comment]
solution :
can create shortcut nvda + ctrl + c for copying last spoken phrase

create nvda + v , which should open last 10 spoken sentences or phrases. there we can select one for copying in to clip board. it is like pressing windows + v .

I think above solution solves most of the problems.

**Comment by josephsl** [comment]
Hi,

A solution does exist somewhat: Speech History add-on. Also, we can't use Control+NVDA+C and NVDA+V for the proposed solution as these commands are already taken.

Thanks.

**Comment by balajip881** [comment]
Thanks for suggesting speech history addion.  This addion solves my problem.
